src/canopy-react-error-boundary.js

```diff
@@ -62,6 +62,16 @@ export default function decorateOptions(opts) {
         }
       }
       componentDidCatch(err, info) {
+        if (info && info.componentStack) {
+          err.extra =  Object.assign(err.extra || {}, {
+            componentStack: info.componentStack
```

Comment:
React adds a `componentStack` property that describes the component where the error originated